tests/common/common_python_vec.py

```diff
@@ -419,37 +482,23 @@ class SwizzleData:
     """)
 
     swizzle_elem_template = Template(
-        """        sycl::vec<${type}, ${size}> inOrderSwizzleFunctionVec {swizzledVec.template swizzle<${in_order_swiz_indexes}>()};
-            if (!check_vector_values<${type}, ${size}>(inOrderSwizzleFunctionVec, in_order_vals)) {
-                resAcc[0] = false;
-            }
-            sycl::vec<${type}, ${size}> reverseOrderSwizzleFunctionVec {swizzledVec.template swizzle<${reverse_order_swiz_indexes}>()};
-            if (!check_vector_values<${type}, ${size}>(reverseOrderSwizzleFunctionVec, reversed_vals)) {
-                resAcc[0] = false;
-            }
-            sycl::vec<${type}, ${size}> inOrderReversedPairSwizzleFunctionVec {swizzledVec.template swizzle<${in_order_reversed_pair_swiz_indexes}>()};
-            if (!check_vector_values<${type}, ${size}>(inOrderReversedPairSwizzleFunctionVec, in_order_reversed_pair_vals)) {
-                resAcc[0] = false;
-            }
-            sycl::vec<${type}, ${size}> reverseOrderReversedPairSwizzleFunctionVec {swizzledVec.template swizzle<${reverse_order_reversed_pair_swiz_indexes}>()};
-            if (!check_vector_values<${type}, ${size}>(reverseOrderReversedPairSwizzleFunctionVec, reverse_order_reversed_pair_vals)) {
-                resAcc[0] = false;
-            }
+        """
+            vecAcc[in_order] = swizzledVec.template swizzle<${in_order_swiz_indexes}>();
+            vecAcc[reverse_order] = swizzledVec.template swizzle<${reverse_order_swiz_indexes}>();
+            vecAcc[in_order_reversed_pair] = swizzledVec.template swizzle<${in_order_reversed_pair_swiz_indexes}>();
+            vecAcc[reverse_order_reversed_pair] = swizzledVec.template swizzle<${reverse_order_reversed_pair_swiz_indexes}>();
     """)
 
     swizzle_full_test_template = Template(
         """        sycl::vec<${type}, ${size}> ${name}DimTestVec = sycl::vec<${type}, ${size}>(${testVecValues});
-            ${type} in_order_vals[] = {${in_order_vals}};
             sycl::vec<${type}, ${size}> inOrderSwizzleFunctionVec {${name}DimTestVec.template swizzle<${in_order_swiz_indexes}>()};
+            vecAcc[in_order] = inOrderSwizzleFunctionVec;
             if (!check_equal_type_bool<sycl::vec<${type}, ${size}>>(inOrderSwizzleFunctionVec)) {
                 resAcc[0] = false;
             }
             if (!check_vector_size<${type}, ${size}>(inOrderSwizzleFunctionVec)) {
                 resAcc[0] = false;
             }
-            if (!check_vector_values<${type}, ${size}>(inOrderSwizzleFunctionVec, in_order_vals)) {
```

Comment:
Note for reviewers: there are two templates for kernel code:
- one for vectors with 4 or less elements, which is used to exercise all simple swizzles like `.xyzw`; modified `swizzle_elem_template` is its part
- another one for bigger vectors, where only per-element swizzles are tested; `swizzle_full_test_template` is used for it

Kernels for both categories of vectors do the same checks to ensure that `.swizzle` method returns elements in the right order, they are just described by two different code templates.